yvalenta/search.rb

```diff
@@ -0,0 +1,55 @@
+class SearchVowels
+
+  def initialize
+    vowels
+  end
+
+  def open_file
+    f = File.open("../text.txt", "r")
+    run(f)
+    f.close
+  end
+
+  def time_start(start)
+    struct = Struct.new(:start)
+    @time_s = struct.new(start)
+  end
+
+  def time_finish(finish)
+    struct = Struct.new(:finish)
+    @time_f = struct.new(finish)
+  end
+
+  private
+
+  def vowels
```

Comment:
I think is better if you define the hash in initialize method, is not necesary to define another method to do it